### ISSUE TITLE
Cannot edit messages on Slack anymore

### ISSUE THREAD

**Issue by Pawely** [meetfranz/franz#1708]
**Describe the bug**
It is not possible to edit messages on slack anymore for me.

**To Reproduce**
1. Write a message on slack
2. Go to the "settings" for this message and click "Edit"
3. Message is not editable

**Expected behavior**
Message should be editable

**Screenshots**

**Desktop (please complete the following information):**
Using Franz with version: 5.4.0-beta.3 

**Additional context**

**Comment by BrendanAtlassian** [comment]
Have you tried https://slack.com/intl/en-nl/help/articles/115004868646-Manage-permissions-for-message-editing-and-deletion- ?

**Comment by rmeffan** [comment]
This just happened to me with the OS X client. Edits still work fine from the same account in the iOS app.

I think it's an issue where there's an edit in progress, but Slack won't tell you where it is. We have dozens of channels (dozens!), so I'm apparently going to have to open every message from the past 24 hours to see if I can find the edit. If that doesn't work, will try nuking prefs, and then give up and try uninstalling / reinstalling.

Bloody annoying stupid bug. If an app is only going to allow a single edit, then you should make sure it's not possible for this sort of UX fail to occur.

**Comment by jeffgerickson** [comment]
I have exactly the same bug.  This is only a problem on the Mac desktop app (both production and beta); I can edit messages in the browser interface (Chrome) and on the iPad app.  Even on the desktop app, the bug appears to be restricted to only one workspace.

I've tried spawning the bug in a second workspace by quitting with an unsaved edit, trying to edit more than one message at a time (which just fails silently), starting a new message while the edit is open, switching channels or workspaces while an edit is unsaved, .... all with no luck.  As @rmeffan suggests, it seems like some internal busy flag has been set incorrectly.

**Comment by Langorithm** [comment]
One of my users had this problem a couple of days ago.
Check your device's time is set correctly and is synced to slack's time settings, that solved it for us.

**Comment by ableuler** [comment]
I had the same problem with the OS X client. Singing out from and re-adding the workspace fixed the problem.

**Comment by novosirj** [comment]
I ran into this recently too, and the same thing fixed it. I'm running Version 4.6.0
1bdb2ca-s@1591717570 (Production). It appeared to be the same cause, and in fact after a reboot, when I tried to switch to a different channel, it asked me if I was prepared to give up the message I was editing. I don't remember what I answered, and it didn't ask me for a second time. There was nothing present in the "Drafts" space in Slack. Attempting to edit a message yielded nothing at all happening. Sometimes, probably if I was in the channel where that stuck message was being edited, clicking Edit would cause the channel to scroll back, but there would be nothing visible for edit.

**Comment by peteroconnor-bc** [comment]
Yep, signed out of company workspace and signed back in - that fixed it for me.

**Comment by peckjon** [comment]
Ditto on sign out/re-add. Thanks for the workaround!

**Comment by cotispear** [comment]
Also encountered this with v4.8.0 on macOS, confirmed that quitting/reopening did **not** resolve the issue, but signing out/back in to my workspace did. 🤷‍♂️

**Comment by jontucichterazo** [comment]
I just had this.
My specific case was that I was editing a pinned post and clicked on a DM to respond to that.
When I clicked back to the pinned post, selecting 'Edit' did nothing.  
Signing out & in again resolved the issue.

I agree with the theory that a 'hidden draft' exists and a bug is preventing it from re-opening.

**Comment by danksim** [comment]
I had the exact same issue as @jontucichterazo on v4.8.0 today. Had to sign out and rejoin workspace.

**Comment by MichaelAroustian** [comment]
> I had the exact same issue as @jontucichterazo on v4.8.0 today. Had to sign out and rejoin workspace.

same here today with version: 4.11.2

**Comment by pcimring-taboola** [comment]
I just hit this issue on my Mac. (The option was there earlier today - and then mysteriously disappeared.)

Slack version 4.11.2.

Signing out and rejoining the workspace has NOT worked for me :(

Any advice?

**Comment by ccooksey** [comment]
I accidentally pinned something in a DM and unpinned it. After that, I could not edit messages. Logging out of the workspace and then back in indeed fixed it.

**Comment by ilyaa-lohika-tix** [comment]
@ccooksey thank you, man!
It helped me too!

**Comment by aryairani** [comment]
Same issue started yesterday, version 4.14.1.  Uninstalling Slack and reinstalling didn't help.  (But also didn't log me out of my workspaces.)

**Comment by levsa** [comment]
Same issue with Version 4.15.0
020384cb0847fd58562308597afb51acc14402a8@1620272744 (Production)

Pressing Up Arrow scrolled the main view for me even though I had focus in the Thread view, so a hidden edit box theory is likely to be true. Logging out of workspace and back in resolved the issue.

**Comment by DBBrowne** [comment]
Same - Logging out of workspace and back in resolved the issue.

**Comment by freshbeka** [comment]
This occurred today - Mac BigSur 11.2.3 and Slack desktop 14.16.2 . Only one workspace would not allow me to Edit (I would click "Edit" and nothing would happen. I was able to edit the exact same message from my iPhone app. This is a rarely used workspace, so I was able to look in each channel and thread and I did not have an edit in process. Solution (Thank you to this issue/thread) was to log out of the workspace and back in. Thanks all!

**Comment by faeb187** [comment]
is it a fact that you cannot edit a "pinned" post? it seems that's not possible..
I'm on newest MacBook (always) with newest OSX (always) and newest Slack (always)

I'm using pinned messages to e.g. write down links and packages, and those require updates over time.
Why can't I update them?
